packages/terra-avatar/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Avatar
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-avatar.svg)](https://www.npmjs.org/package/terra-avatar)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+The terra-avatar component displays an avatar which can be either an image or text. If neither an image or text is provided, a variant-based fallback icon is used. The avatar displays in a circular wrapper.
+
+- [Getting Started](#getting-started)
+- [Documentation](https://github.com/cerner/terra-core/tree/master/packages/terra-avatar/docs)
+- [LICENSE](#license)
+
+## Getting Started
+
+- Install from [npmjs](https://www.npmjs.com): `npm install terra-avatar`
+
+## LICENSE
+
+Copyright 2017 Cerner Innovation, Inc.
```

Comment:
I think we need a 2018 here? I also think the latest terra generator fixes this as well.

Reply:
Updated in this [commit](https://github.com/cerner/terra-core/commit/855d5fc5c14ce5dc2a29cb07d7d5575af2320962).